src/content/configuration/dev-server.md

```diff
@@ -66,6 +67,8 @@ W> You cannot use the second `compiler` argument (a callback) when using `Webpac
 
 W> Be aware that when [exporting multiple configurations](/configuration/configuration-types/#exporting-multiple-configurations) only the `devServer` options for the first configuration will be taken into account and used for all the configurations in the array.
 
+T > dev-server uses `output.path` to be the root path for serving generated files. For the output config `{ filename: 'bundle.js', path: path.resolve(__dirname, 'assets'), }`, bundle.js will be available under localhost:[port]/bundle.js instead of localhost:[port]/assets/bundle.js. Adjust output config to `{ filename: 'assets/bundle.js', path: path.resolve(__dirname), }` if you want to serve generated files under localhost:[port]/assets/bundle.js.
+
```

Comment:
I think we should clarify `publicPath` here otherwise it some misleading

Reply:
Maybe we should mention both solutions? If the publicPath solution is adopted, the logic will be required to be defined duplicately, in both `output.path` and `devServer.publicPath`.

Reply:
`webpack-dev-server@4` respect `output.publicPath`, but I think we should clarify:
- output.path + output.filename (i.e. `output.publicPath` is empty for webpack@4 and `auto` for webpack@5)
- output.path + output.publicPath + output.filename

Reply:
I have rewritten the paragraph. Is it correct and OK?